Reference/Searching/Examine/Indexing/index-v8.md

```diff
@@ -298,6 +298,9 @@ public class ProductIndexCreator : LuceneIndexCreator, IUmbracoIndexesCreator
 }
 ```
 
+You can create an `IValueSetBuilder` implementation that builds the value sets for the index following the example [here](index.md/productindexvaluesetbuilder).
+You can also create an `IndexPopulator` implementation that populates the index with the value sets following the example [here](index.md/productindexpopulator).
```

Comment:
We prefer to not use "here" as link texts, so I've tried to amend your suggestions a bit.
Let me know what you think!

```suggestion
You can create an `IValueSetBuilder` implementation that builds the value sets for the index following the [example for Umbraco 9](index.md/#productindexvaluesetbuilder).
You can also create an `IndexPopulator` implementation that populates the index with the value sets following the [example for Umbraco 9](index.md/#productindexpopulator).
```